utils/timezone.jsx

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import UserStore from 'stores/user_store.jsx';
+import * as UserActions from 'actions/user_actions.jsx';
+
+const dateTimeFormat = new Intl.DateTimeFormat();
+
+export function getSupportedTimezones() {
+    if (global.mm_config && global.mm_config.SupportedTimezones) {
+        return global.mm_config.SupportedTimezones.split(',');
+    }
+    return [];
+}
+
+export function getBrowserTimezone() {
+    return dateTimeFormat.resolvedOptions().timeZone;
```

Comment:
Did some quick tests with [this JSFiddle](https://jsfiddle.net/revpjacv/1/) and IE 11.15 displays "undefined" for this JS API so we'll need a fallback to support it.

@jasonblais Can you please confirm/disconfirm support for IE 11.15?

![latest-ie11](https://user-images.githubusercontent.com/1149597/36996910-f808d008-2085-11e8-8d73-c6630072122b.png)

Reply:
Thanks Martin. I had missed your note until now. I'll queue for our PM meeting to discuss this.

Reply:
Discussed and we officially support IE11, hence IE 11.15 is included